confgenerator/testdata/valid/linux/metrics-receiver_kafka/input.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
+  receivers:
+    kafkametrics:
+      type: kafka
+      # exercising default endpoint
+      username: otelu
+      password: otelp
+      collection_interval: 30s
+  service:
+    pipelines:
+      kafkapipeline:
+        receivers:
+          - kafkametrics
```

Comment:
It's preferred for the receiver and pipeline IDs to stay the same as the receiver type in the sample configuration. E.g.  "s/{APP}_metrics/{APP}", "s/{APP}_pipeline/{APP}", "s/{APP}_default/{APP}"

Some users look at the `valid` cases for sample configurations as well. So the main `confgenerator/testdata/valid/{LINUX_OR_WINDOWS}/metrics-receiver_{APP}/input.yaml` test case should try follow this convention as well.

Reply:
Looks like it's still `kafkametrics` and `kafkapipeline`. Is there a pending commit that has not yet been pushed?

Reply:
Sent a suggestion